scripts/gn/templates/auto_template.yaml

```diff
@@ -0,0 +1,319 @@
+inputs:
+
+    inputs_root: "."
+
+    atx_files:  [ ../../inputData/products/igs20.atx                    ] # required
+    egm_files:  [ ../../inputData/products/tables/EGM2008.gfc           ] # Earth gravity model coefficients file
+    igrf_files: [ ../../inputData/products/igrf13coeffs.txt             ]
```

Comment:
Should be `../../inputData/products/tables/igrf13coeffs.txt` (i.e. in the tables directory)